docs/userguide.md

```diff
@@ -57,17 +56,17 @@
 
 # User Guide
 
-Please refer to [Kubefed Concepts](./concepts.md) first before you go through this user guide.
+Please refer to [KubeFed Concepts](./concepts.md) first before you go through this user guide.
 
-This user guide contains concepts and procedures to help you get started with Kubefed.
+This user guide contains concepts and procedures to help you get started with KubeFed.
 
-For information about installing Kubefed, see the [installation documentation](./installation.md).
+For information about installing KubeFed, see the [installation documentation](./installation.md).
 
-## Joining and unjoining clusters
+**NOTE:** The features described in this documentation are in Alpha state and are not yet supported. KubeFed should not be used in a production environment.
```

Comment:
@abrennan89 I'm assuming you have a good reason for adding this, but some features described in the document will be in beta soon and supported.  Please update accordingly in a future PR.

Reply:
Only added this because you had an issue about calling out the status - removed the note, we can chat offline about what to do with this :)